### ISSUE TITLE
[Ingest-Core-UI] Multiple Submission MVP: Do not accept new submission for project if an open submission already exists.

### ISSUE THREAD

**Issue by MightyAx** [HumanCellAtlas/ingest-central#605]
This enhancement will resolve the issues of #583 isolating multiple concurrent submissions for a project, by denying attempts to make submissions for projects that have incomplete submissions.
Implementing [this MVP](https://github.com/HumanCellAtlas/ingest-central/issues/583#issuecomment-538367670)

For this feature to make sense, it is dependent on #582 so that creating a project does not require a submission and #599 so that users can delete submissions that contain errors.

This will require some changes in ingest-core to deny submissions at an API level and ingest-ui to show the errors in a way that makes sense to users.

**Comment by MightyAx** [comment]
this ticket has been superseded by #627 & #628
